docs/Release_notes.md

```diff
@@ -1,6 +1,67 @@
 CARMA System Release Notes
 ----------------------------
 
+Version 4.8.0, released Sep 11th, 2024
+----------------------------------------
+
+### **Summary**
+This release introduces the CARMA 1Tenth (C1T) software platform with a focus on being able to perform an automated Port Drayage demonstration within a scaled-down test environment. The system is built around a forked version of the Navigation2 (Nav2) ADS platform, with the new navigation2_extensions repository developed to integrate Cooperative Automated Driving (CDA) functionality with Nav2. A forked version of the vesc repository and the newly-developed twist_to_ackermann repository were introduced to allow for minor changes needed to interface software drivers with the C1T hardware. Additional SQL files were added to V2X Hub to coordinate the Port Drayage demonstration message exchange with the vehicle, and the c1t2x-emulator package was developed to enable this communication between two Raspberry Pis representing mock OBUs and RSUs. Minor changes were made to the  carma-msgs, carma-utils, and carma-messenger repositories to integrate with the C1T system and ensure build compatibility with ARM platforms and ROS 2 Humble. Lastly, the c1t_bringup repository was developed to coordinate launching the necessary software onboard the vehicle with the specified parameters, maps, and road networks.
+
+### **C1t2x-emulator** 
+c1t2x-emulator is a new repository that enables Raspberry Pi’s to act as mock Onboard Units (OBUs) and Roadside Units (RSUs) for the C1T platform. This includes the ability to communicate with vehicles and V2X Hub instances, as well as forwarding messages over WiFi to other Raspberry Pi’s. 
+
+NOTE: A known issue with this release is packets being sent between Raspberry Pi's are occasionally dropped, causing the automated Port Drayage demonstration to hang while the vehicle waits for a new instruction that it will never receive. This issue will be addressed in the following release.
```

Comment:
@SaikrishnaBairamoni  Please follow the format we've done before for Known Issues. Per John C. they have have  a story to resolve it in
[CF-928](https://usdot-carma.atlassian.net/browse/CF-928).

Reply:
Updated